packages/augur-sdk/src/state/db/MarketDB.ts

```diff
@@ -19,6 +18,23 @@ import { MarketData, MarketType, OrderType } from "../logs/types";
 import { BigNumber } from "bignumber.js";
 import { Orderbook } from "../../api/Liquidity";
 
+// because flexsearch is a UMD type lib
+const flexSearch = require('flexsearch');
+import { Index, ExtendedSearchOptions, SearchResults } from 'flexsearch';
+
+export interface MarketFields {
+  id: string;
+  market: string;
+  category1: string;
+  category2: string;
+  category3: string;
+  description: string;
+  longDescription: string;
+  resolutionSource: string;
+  _scalarDenomination: string;
+  start: Date;
+  end: Date;
```

Comment:
can we add market creator address, just in case users want to search on it, to see all the markets created by a address.

Reply:
Added to https://github.com/AugurProject/augur/issues/3023